### ISSUE TITLE
RTX50系列研究中

### ISSUE THREAD

**Issue by Astin1860** [GuijiAI/HeyGem.ai#316]
RTX5070TI，已经重新构建了CUDA128+PYTORCH2.8的暗夜版镜像，但还是不成功，原因可能是主体代码用PY3.8编译的原因，在调用CUDA的时候就会出错。目前导出了一份需求文件，有兴趣的同学可以试试移植到COMFYUI里
cv2box==0.5.9
apstone==0.0.8
appdirs==1.4.4
audioread==2.1.9
cffi==1.15.0
charset-normalizer==2.0.12
click==8.1.3
colorama==0.4.4
cycler==0.11.0
decorator==5.1.1
filelock==3.7.1
flatbuffers==2.0
fonttools==4.36.0
freetype-py==2.3.0
huggingface-hub==0.0.8
idna==3.3
imageio==2.19.3
importlib-metadata==4.11.4
joblib==1.1.0
kiwisolver==1.4.4
kornia==0.6.6
librosa==0.8.1
llvmlite==0.38.1
matplotlib==3.5.3
networkx==2.6.3
numba==0.55.2
numpy==1.21.6
nvdiffrast==0.2.5
onnxruntime-gpu==1.9.0
opencv-python==4.7.0.72
packaging==21.3
pillow==9.1.1
pooch==1.6.0
protobuf==4.21.5
psutil==5.9.1
pycparser==2.21
pyglet==1.5.26
pyopengl==3.1.0
pyparsing==3.0.9
pyrender==0.1.45
python-dateutil==2.8.2
pywavelets==1.3.0
pyyaml==6.0
regex==2022.6.2
requests==2.27.1
resampy==0.2.2
sacremoses==0.0.53
scikit-image==0.19.3
scikit-learn==1.0.2
scipy==1.7.1
six==1.16.0
soundfile==0.10.3.post1
threadpoolctl==3.1.0
tifffile==2021.11.2
tokenizers==0.10.3
torch==1.11.0+cu113
torchaudio==0.11.0+cu113
torchvision==0.12.0+cu113
tqdm==4.64.0
transformers==4.6.1
trimesh==3.12.7
typeguard==2.13.3
typing-extensions==4.2.0
urllib3==1.26.9
zipp==3.8.0

**Comment by Tianlucy** [comment]
50系解决了吗

**Comment by xiaotang-12-ops** [comment]
50系解决了吗，好想用了...

**Comment by Astin1860** [comment]
> 50系解决了吗，好想用了...

解决不了，除非把项目原码用PY310重写一下。原因是项目是在PY38的基础上写的。现在RTX50可以支持的pytorch版本2.8，最低适配PY39的环境。我单独在Linux上用基础镜像手动配置环境，或是英伟达12.8的镜像，部署2个版本的py,用py310来配置pytorch，用PY38来做别的依赖部署，把项目手动部署，能勉强启动，但是会报错。原项目的核心代码没有用PY39及以上的版本重写之前，等于无解。

**Comment by xiaotang-12-ops** [comment]
> > 50系解决了吗，好想用了...
> 
> 解决不了，除非把项目原码用PY310重写一下。原因是项目是在PY38的基础上写的。现在RTX50可以支持的pytorch版本2.8，最低适配PY39的环境。我单独在Linux上用基础镜像手动配置环境，或是英伟达12.8的镜像，部署2个版本的py，用py310来配置pytorch，用PY38来做别的依赖部署，把项目手动部署，能勉强启动，但是会报错。原项目的核心代码没有用PY39及以上的版本重写之前，等于无解。

好的谢谢回答！
